src/uct/ib/base/ib_device.h

```diff
@@ -409,6 +409,10 @@ ucs_status_t uct_ib_device_get_roce_ndev_name(uct_ib_device_t *dev,
                                               uint8_t gid_index,
                                               char *ndev_name, size_t max);
 
+ucs_status_t
+uct_ib_device_get_roce_ndev_index(uct_ib_device_t *dev, uint8_t port_num,
+                                  int gid_index, int *iface_index_p);
```

Comment:
`iface_index_p` -> `ndev_index_p`, for 2 reasons:
1. consistency w above.
2. when a func is called `get_X`, it is reasonable to expect `X` (or similar) in args